database/seeders/DatabaseSeeder.php

```diff
@@ -12,13 +15,21 @@ class DatabaseSeeder extends Seeder
      *
      * @return void
      */
-    public function run()
-    {
-        // \App\Models\User::factory(10)->create();
+    public function run(){
+        Category::create(['name' => 'Card Game','category_code' => 'CARD']);
+        Category::create(['name' => 'Economic','category_code' => 'ECON']);
+        Category::create(['name' => 'Negotiation','category_code' => 'NEGO']);
+        Category::create(['name' => 'Dice','category_code' => 'DICE']);
+        Category::create(['name' => 'Adventure','category_code' => 'ADVN']);
+        Category::create(['name' => 'Fantasy','category_code' => 'FANT']);
+        Category::create(['name' => 'Fighting','category_code' => 'FIGH']);
+        Category::create(['name' => 'Puzzle','category_code' => 'PUZZ']);
+        Category::create(['name' => 'Miniatures','category_code' => 'MINI']);
+        Category::create(['name' => 'Territory Building','category_code' => 'TERR']);
+        Category::create(['name' => 'War Game','category_code' => 'WARG']);
+        Category::create(['name' => 'Science Fiction','category_code' => 'SCIF']);
+        Category::create(['name' => 'Civilization','category_code' => 'CIVI']);
```

Comment:
Okay so this all looks good to me, but I think this should go in the CategorySeeder unless I am mistaken? Which could because I am mistaken a lot haha. From reading the issue itself, I didn't see anything about needing to edit the Database Seeder but I could be missing something. I would defer to @oldskoolfan to confirm.

Reply:
I think you're right about that, I can change it once I'm back Sunday night

Reply:
yes, this should go in the CategorySeeder